src/kixi/heimdall/pwd_resets.clj

```diff
@@ -0,0 +1,61 @@
+(ns kixi.heimdall.pwd-resets
+  (:require [taoensso.timbre :as log]
+            [clojure.spec :as s]
+            [kixi.heimdall.components.database :as db]
+            [kixi.heimdall.util :as util]
+            [kixi.heimdall
+             [schema :as schema]
+             [util :as util]]))
+
+(def resets-table "pwd-resets")
```

Comment:
I realize this is bikeshedding but it's better to use the full name to reduce cognitive load :)